### ISSUE TITLE
Validation Loss

### ISSUE THREAD

**Issue by Vikram12301** [codetiger/ThiruvalluvarGPT#4]
**Is there any reason on why the validation loss first decreased drastically and then started increasing?**

![Capture](https://user-images.githubusercontent.com/62374169/229016322-3126b5be-3fff-4d9e-bd30-81ba837bc883.PNG)

**Comment by codetiger** [comment]
My understanding is, the dataset is too small for the model. If you train too much with training dataset, beyond a point the validation data set does not fit well. In this use case, each kural is very unique, so the model's optimal learning steps is around 1000 for this dataset. 

What I learnt recently is, this effect is call overfitting.

Note: Am not a professional in ML or Statistics so take my response lite.
